plasmapy/atomic/atomic.py

```diff
@@ -1,60 +1,37 @@
-"""Functions that retrieve or are related to elemental or isotopic data."""
+r"""Functions that retrieve or are related to elemental or isotopic data."""
```

Comment:
I ran into some problems a while back with doc builds that I had trouble pinpointing, and it turned out that the issue was that there was LaTeX in there but I didn't mark it as a raw docstring.  There were a few other occasions where I added math to a docstring and forgot to put in the `r`, which led to errors that caused problems.  I have been using raw docstrings pretty diligently because if an inexperienced contributor adds an equation to a docstring where there wasn't one before, then they won't run into the same problem.  Plus, if inexperienced contributors see `r"""` at the beginning of docstrings, they may get into that habit without really knowing why, and thus avoid this problem automatically should they write a new docstring with math in it.  Admittedly, doing this for one-line docstrings with no chance of math being added might be too much!